src/main/java/com/xiao/users/dto/UserDto.java

```diff
@@ -1,13 +1,16 @@
 package com.xiao.users.dto;
 
+import com.xiao.users.validator.CheckAllValueEmpty;
 import jakarta.validation.constraints.Email;
 import jakarta.validation.constraints.NotEmpty;
 import jakarta.validation.constraints.Size;
 import lombok.Builder;
 import lombok.Data;
 
+
 @Data
 @Builder
+@CheckAllValueEmpty(fields = {"username", "password", "emailAddress"}, message = "Enter at least 1 piece of information")
```

Comment:
The UserDto was validated by @ NotEmpty, we dont need the @CheckAllValueEmpty here